scripts/ghutils.py

```diff
@@ -436,19 +435,32 @@ def create_release(repo, version, bodyfile):
     # Store version in context for use in notifications
     ctx_obj.version = version
     try:
-        release = repo.create_release(
-            tag_name=version,
-            name=version,
-            body=release_notes,
-        )
-        logger.info("Release {} created.".format(version))
+        try:
+            release = [r for r in repo.iter_releases() if r.name == version][0]
+            if release.body != release_notes:
+                raise ReleaseFailureException(
```

Comment:
If someone has attempted to release the wrong SHA and the updates it, how would that be addressed?